test/index.spec.js

```diff
@@ -51,3 +96,29 @@ test('named export | throw because wrong value type (undefined)', () => {
 
   expect(result).toThrow('value has to be typeof: \'string\' but got typeof: \'undefined\'');
 });
+
+
+test('named export | throw because wrong value type (undefined)', () => {
+  const input = undefined;
+
+  const result = () => convertStringToNumber(input);
+
+  expect(result).toThrow('value has to be typeof: \'string\' but got typeof: \'undefined\'');
```

Comment:
Just to keep it consistent, store the expected value as a constant.

Reply:
done.